### ISSUE TITLE
question about the result

### ISSUE THREAD

**Issue by RuiWang-coder** [lingyanruan/DPDNet#1]
Hello ruan!
First, thanks to your retrained codes and weight.
I have some question when i test use your code and pre-trained weight, I use three RTX3090, when testing, all three cards are full of video memory, and the result is not satisfactory, psnr is only 9.1!!! The result images are blur, red or green images.
Did I do something wrong? And I was wondering how many gpu did you used at test stage? 
Looking forward to your reply.

**Comment by lingyanruan** [comment]
Hi,

I have just re-tested the code and it did well on my side.
I use single Tesla V100 with 32G to test the code, and the results are the same as we reported in the paper.

Did you change any settings? I am not sure if it is related to data range?

**Comment by RuiWang-coder** [comment]
I didn't change anything, I just use the test set opend by [https://github.com/Abdullah-Abuolaim/defocus-deblurring-dual-pixel](url), it's that ok?

**Comment by RuiWang-coder** [comment]
Thanks to your reply!
I didn't change anything, I just use the test set opend by [https://github.com/Abdullah-Abuolaim/defocus-deblurring-dual-pixel](url), it's that ok?
 And i was wondering ,whatever how many cards i used(1,or2,or3),  the cards are full of video memory, why that happend?

**Comment by RuiWang-coder** [comment]
I was wondering if its the weight problem? Could you please re-upload your final weight?

**Comment by lingyanruan** [comment]
My re-test is exactly using the shared weight and no issue occurred.
